core/src/SYCL/Kokkos_SYCL_Instance.hpp

```diff
@@ -356,3 +357,4 @@ struct sycl::is_device_copyable<
     : std::true_type {};
 #endif
 #endif
+#endif
```

Comment:
We don't want
```C++
template <typename Functor, typename Storage>
struct sycl::is_device_copyable<
    const Kokkos::Experimental::Impl::SYCLFunctionWrapper<Functor, Storage,
                                                          false>>
    : std::true_type {};
```
any more?

Reply:
Correct. The whole point of this workaround was to figure out if a `const`-qualified type is implicitly device-copyable if the non-`const` type is since this behavior changed between `oneAPI` versions where the latest versions don't allow using `SFINAE` anymore and do the "right thing" anyway.